definitions/npm/prismic.io_v3.x.x/flow_>=v0.28.x/prismic.io_v3.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module 'prismic.io' {
-  declare type linkResolver = (...args: any) => any;
+  declare type LinkResolver = (...args: any) => any;
```

Comment:
James is right about this, and in almost every case I would agree, howeveri would be surprised if anyone but myself was using this definition file (I originally submitted it). Without the types that I expose in this PR the definition isn't very useful. There was also a bug in one of the major API endpoints of this definition. 
I have learnt that I need to give my defs more usage time before submitting.